chain-registry/src/utils.rs

```diff
@@ -0,0 +1,268 @@
+    "cosmoshub-impacthub.json",
+    "cosmoshub-irisnet.json",
+    "cosmoshub-juno.json",
+    "cosmoshub-likecoin.json",
+    "cosmoshub-osmosis.json",
+    "cosmoshub-persistence.json",
+    "cosmoshub-regen.json",
+    "cosmoshub-secretnetwork.json",
+    "cosmoshub-sentinel.json",
+    "cosmoshub-sifchain.json",
+    "cosmoshub-starname.json",
+    "cosmoshub-umee.json",
+    "crescent-gravitybridge.json",
+    "crescent-osmosis.json",
+    "crescent-secretnetwork.json",
+    "cryptoorgchain-irisnet.json",
+    "cryptoorgchain-osmosis.json",
+    "cryptoorgchain-persistence.json",
+    "cryptoorgchain-regen.json",
+    "cryptoorgchain-sentinel.json",
+    "cryptoorgchain-sifchain.json",
+    "cryptoorgchain-starname.json",
+    "decentr-osmosis.json",
+    "desmos-osmosis.json",
+    "dig-juno.json",
+    "dig-osmosis.json",
+    "emoney-irisnet.json",
+    "emoney-juno.json",
+    "emoney-osmosis.json",
+    "evmos-osmosis.json",
+    "evmos-secretnetwork.json",
+    "fetchhub-osmosis.json",
+    "galaxy-osmosis.json",
+    "genesisl1-osmosis.json",
+    "gravitybridge-osmosis.json",
+    "gravitybridge-secretnetwork.json",
+    "impacthub-osmosis.json",
+    "impacthub-sifchain.json",
+    "injective-osmosis.json",
+    "injective-secretnetwork.json",
+    "irisnet-osmosis.json",
+    "irisnet-persistence.json",
+    "irisnet-regen.json",
+    "irisnet-sentinel.json",
+    "irisnet-sifchain.json",
+    "irisnet-starname.json",
+    "juno-osmosis.json",
+    "juno-persistence.json",
+    "juno-secretnetwork.json",
+    "juno-sifchain.json",
+    "juno-stargaze.json",
+    "juno-terra.json",
+    "kava-osmosis.json",
+    "kichain-osmosis.json",
+    "konstellation-osmosis.json",
+    "kujira-osmosis.json",
+    "likecoin-osmosis.json",
+    "lumenx-osmosis.json",
+    "lumnetwork-osmosis.json",
+    "meme-osmosis.json",
+    "microtick-osmosis.json",
+    "oraichain-osmosis.json",
+    "osmosis-panacea.json",
+    "osmosis-persistence.json",
+    "osmosis-provenance.json",
+    "osmosis-regen.json",
+    "osmosis-rizon.json",
+    "osmosis-secretnetwork.json",
+    "osmosis-sentinel.json",
+    "osmosis-shentu.json",
+    "osmosis-sifchain.json",
+    "osmosis-sommelier.json",
+    "osmosis-stargaze.json",
+    "osmosis-starname.json",
+    "osmosis-terra.json",
+    "osmosis-tgrade.json",
+    "osmosis-umee.json",
+    "osmosis-vidulum.json",
+    "persistence-regen.json",
+    "persistence-sentinel.json",
+    "persistence-sifchain.json",
+    "persistence-starname.json",
+    "regen-sentinel.json",
+    "regen-sifchain.json",
+    "regen-starname.json",
+    "secretnetwork-sentinel.json",
+    "secretnetwork-sifchain.json",
+    "secretnetwork-stargaze.json",
+    "secretnetwork-terra.json",
+    "secretnetwork-terra2.json",
+    "sentinel-sifchain.json",
+    "sentinel-starname.json",
+];
+
+/// `Fetchable` represents the basic expectations for external data or resources that
+/// can be fetched.
+#[async_trait]
+pub trait Fetchable
```

Comment:
I think it would make more sense to move the `Fetchable` trait to its own file `chain-registry/src/fetchable.rs`.